packages/react-native-sdk/src/components/participants/ParticipantView.tsx

```diff
@@ -0,0 +1,276 @@
+import React, { ComponentType, useEffect, useRef } from 'react';
+import { StyleProp, StyleSheet, View, ViewStyle } from 'react-native';
+import {
+  CallingState,
+  SfuModels,
+  StreamVideoParticipant,
+  VisibilityState,
+} from '@stream-io/video-client';
+import { useCall, useCallCallingState } from '@stream-io/video-react-bindings';
+import { Z_INDEX } from '../../constants';
+import { theme } from '../../theme';
+import { palette } from '../../theme/constants';
+import {
+  ParticipantNetworkQualityIndicator as DefaultParticipantNetworkQualityIndicator,
+  ParticipantNetworkQualityIndicatorProps,
+} from './ParticipantNetworkQualityIndicator';
+import {
+  ParticipantReaction as DefaultParticipantReaction,
+  ParticipantReactionProps,
+} from './ParticipantReaction';
+import {
+  ParticipantLabel as DefaultParticipantLabel,
+  ParticipantLabelProps,
+} from './ParticipantLabel';
+import {
+  ParticipantVideo as DefaultParticipantVideo,
+  ParticipantVideoProps,
+} from './ParticipantVideo';
+import { ParticipantVideoPlaceholderProps } from './ParticipantVideoPlaceholder';
+
+export type ParticipantVideoType = 'video' | 'screen';
+
+/**
+ * Props to be passed for the Participant component.
+ */
+export type ParticipantViewProps = {
```

Comment:
we need a style prop here too for overriding the container style

Reply:
Styling is for later, as I mentioned in my PR description and this goes hand in hand with theming.